deriv.com.conf

```diff
@@ -32,7 +32,26 @@ server {
     listen 80;
     server_name  _;
 
-    add_header Cache-Control "public, max-age=7200, s-maxage=600";
+    location ~* \.html$ {
+        add_header Cache-Control "public, no-cache, must-revalidate";
+    }
+
+    location /page-data {
+        add_header Cache-Control "public, no-cache, must-revalidate";
+    }
+
+    location = /sw.js {
```

Comment:
is sw.js service worker related?

Reply:
yes we never want to cache the service worker - as we it should be revalidated upon each load to check if a new version of the site is available. (once we add the service worker that is :) )